### ISSUE TITLE
This rule still needed?

### ISSUE THREAD

**Issue by furt555nv** [AdguardTeam/AdguardFilters#94222]
Cookie notification on these sites is already blocked by the general rule `##.bs-wrap-gdpr-law`, so it looks like rule on line 2369 is no longer needed.

https://github.com/AdguardTeam/AdguardFilters/blob/2507a75c4f3d4acf2bc271fa0efc2d03d7911a1f/AnnoyancesFilter/sections/cookies_specific.txt#L2369
